common/src/main/java/io/netty/util/internal/UnpaddedInternalThreadLocalMap.java

```diff
@@ -13,49 +13,11 @@
  * License for the specific language governing permissions and limitations
  * under the License.
  */
-
 package io.netty.util.internal;
 
-import io.netty.util.concurrent.FastThreadLocal;
-
-import java.nio.charset.Charset;
-import java.nio.charset.CharsetDecoder;
-import java.nio.charset.CharsetEncoder;
-import java.util.ArrayList;
-import java.util.Map;
-import java.util.concurrent.atomic.AtomicInteger;
-
 /**
- * The internal data structure that stores the thread-local variables for Netty and all {@link FastThreadLocal}s.
- * Note that this class is for internal use only and is subject to change at any time.  Use {@link FastThreadLocal}
- * unless you know what you are doing.
+ * @deprecated This class will be removed in the future.
  */
 class UnpaddedInternalThreadLocalMap {
```

Comment:
can't we just remove this class as its package-private now that we moved the fields ?

Reply:
No, I get compatibility errors when I do.

Reply:
bummer... I think this is a false positive. Can you report it https://github.com/siom79/japicmp ?

Reply:
Opened an issue about it: https://github.com/siom79/japicmp/issues/277